ui/app/components/tools/wrap.ts

```diff
@@ -34,11 +40,11 @@ export default class ToolsWrap extends Component {
     // otherwise the following codemirror modifier check will pass `this._editor.getValue() !== namedArgs.content` and _setValue will be called.
     // the method _setValue moves the cursor to the beginning of the text field.
     // the effect is that the cursor jumps after the first key input.
-    return JSON.stringify({ '': '' }, null, 2);
+    return stringify([{ '': '' }], { skipFormat: false });
```

Comment:
It took me a while to understand where skipFormat was coming from (our own stringify util). My preference is we don't use that pattern but what was already there. It obscures what's happening. That helper is very old and I'd prefer native javascript. Thoughts? Note: we use the code on previous line 37 in other places as well and it would be confusing why we are not continuing the pattern here.

Reply:
Ya I thought about that but felt like it was getting out of scope for this PR. There are 25 uses of the helper in template files so I decided to update this file to use the helper in both places for consistency. I could update it to both use `JSON.stringify` instead though.

Reply:
If you could keep JSON.stringify that would be my preference. Specifically for handling this use in the codemirror. It will make replacing the codemirror instances a little less "why is this one this way" for any dev replacing codemirror (this pattern is a copy/paste for the other instances).

Reply:
I'm going to leave as is for now since there are a couple of other components using the helper. It might be worth making a ticket to remove the instances and maybe add a comment to the helper to only use in templates if that's the way we want to go.